src/wx/wxvbam.h

```diff
@@ -397,6 +399,7 @@ class JoystickPoller : public wxTimer {
         void Notify() {
             wxGetApp().frame->PollJoysticks();
         }
+        // This isn't called for dialogs but actually only for main window. Is this intended?
```

Comment:
The previous implementation only used this for dialogs, since the OnIdle loop in the panel does the polling itself, feel free to change the method name to `ShowWindow` or some such. That would be clearer.